release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-// 4.14.29
+[id="ocp-4-14-30"]
+=== RHSA-2024:3881 - {product-title} 4.14.30 bug fix and security updates
+
+Issued: 2024-06-19
+
+{product-title} release 4.14.30, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:3881[RHSA-2024:3881] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3918[RHSA-2024:3918] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.30 --pullspecs
+----
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously,  (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
+
+* https://issues.redhat.com/browse/OCPBUGS-35034
+
+* Previously, an AWS policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
+
+* https://issues.redhat.com/browse/OCPBUGS-34814
+
+* https://issues.redhat.com/browse/OCPBUGS-34792 RHE
+
+* https://issues.redhat.com/browse/OCPBUGS-34668
+
+* https://issues.redhat.com/browse/OCPBUGS-34657
```

Comment:
* Previously, the 'metal3-ironic' and 'metal3-ironic-inspector' pods failed when upgrading to {product-title} 4.15.11 from 4.15.8 due to an install failure related to FIPS mode enablement. With this release, the issue has been resolved. (link:[https://issues.redhat.com/browse/OCPBUGS-34657[OCPBUGS-34657]).